ihatemoney/web.py

```diff
@@ -363,12 +363,24 @@ def remind_password():
             if success:
                 return redirect(url_for(".password_reminder_sent"))
             else:
+                # Only display admin email if its not the default and public creation is allowed
+                admin_email = current_app.config.get("MAIL_DEFAULT_SENDER")
+                error_extension = ""
+                if (
+                    admin_email
+                    and admin_email[1] != "budget@notmyidea.org"
+                    and current_app.config.get("ALLOW_PUBLIC_PROJECT_CREATION")
```

Comment:
This check is for only adding the email address to the warning message, not for sending the whole message. From my understanding, this was part of the issue #626:

> don't do that if public project creation is forbidden (because it would reveal more information to unauthentified users)

Maybe I misunderstood or am using the wrong variable?

Reply:
Ha okay, I understand better. I feel this is more a side effect than the correct usage of this variable. We probably should add another configuration variable for this, no ?